rdkit/Chem/EState/UnitTestVSA.py

```diff
@@ -12,52 +12,44 @@
 
 """
 from __future__ import print_function
-from rdkit import RDConfig
-import unittest, os
+
+import os
+import unittest
+
 from rdkit import Chem
 from rdkit.Chem.EState import EState_VSA
-import os.path
 
 
-def feq(n1, n2, tol=1e-4):
-  return abs(n1 - n2) <= tol
+class TestCase(unittest.TestCase):
 
+  @staticmethod
```

Comment:
That's a technical distinction.  However python has no real "private static method" and one may reasonably assume that the method is usable without the rest of the class which I don't really think is the case here.  Not that it really matters in this case and it is a purely stylistic concept but I tend to prefer static methods to define external api points, not internal ones.